back/src/companies/resolvers/mutations/__tests__/verifyCompanyByAdmin.integration.ts

```diff
@@ -24,7 +24,7 @@ const VERIFY_COMPANY_BY_ADMIN = gql`
 `;
 
 describe("mutation verifyCompanyByAdmin", () => {
-  afterAll(resetDatabase);
+  afterEach(resetDatabase);
```

Comment:
Je ne sais pas pourquoi ce test s'est mis à échouer, la Company était créé quand elle ne devait pas donc j'ai mis un reset après chaque `it`